docs/source/developers/python.rst

```diff
@@ -80,6 +80,16 @@ run
 
 and look for the "custom options" section.
 
+.. note::
+
+   There are a few low-level tests written directly in C++ for historical
+   reasons. These tests are also part of the ``pytest``-based test suite and
```

Comment:
Not only for historical reasons, because they also test low-level internals such as `OwnedRef`.